ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/PicoCliWeb3ProviderTrustStore.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.config;
+
+import tech.pegasys.ethsigner.core.config.PkcsStoreConfig;
+
+import java.io.File;
+
+import picocli.CommandLine.Option;
+
+public class PicoCliWeb3ProviderTrustStore implements PkcsStoreConfig {
```

Comment:
web3provider vs downstream - do they mean the same thing?

Reply:
yeah same thing, renamed to ...Downstream...